TYPO3.Flow/Documentation/TheDefinitiveGuide/PartV/CodingGuideLines/PHP.rst

```diff
@@ -576,27 +494,24 @@ The syntax is as follows:
    space. This format is rendered correctly by Forge (redmine)
  * Use a hanging indent
 
- FLOW-123 #close
- FLOW-789 #comment Fixes this partly, needs more work
+ Fixes #123
 
 
 Examples of good and bad subject lines::
 
-   Introduce xyz service                            // BAD, missing code prefix
-   BUGFIX: Fixed bug xyz                            // BAD, subject should be written in present tense
-   WIP !!! TASK: A breaking change                  // BAD, subject has to start with [!!!] for breaking changes
-   BUGFIX: SessionManager removes expired sessions  // GOOD, the line explains what the change does, not what the
-                                                       bug is about (this should be explained in the following lines
-                                                       and in the related bug tracker ticket)
+   Introduce xyz service                               // BAD, missing code prefix
+   BUGFIX: Fixed bug xyz                               // BAD, subject should be written in present tense
+   WIP !!! TASK: A breaking change                     // BAD, subject has to start with [!!!] for breaking changes
+   BUGFIX: SessionManager will remove expired sessions // GOOD, the line explains what the change does, not what the
```

Comment:
Because the former did not really be clear about what the bug or fix is. Was the bug that sessions are removed or the opposite? But the new version is still not really køøl, true.